RELEASES.md

```diff
@@ -195,51 +195,60 @@ framework.
 ### Installing PyTorch Python packages
 
 > [!WARNING]
-> This is under **active** development.
+> This is under **active** development. Compatibility matrix for these wheels
+> is in progress.
 
 Using the index pages listed above, you can install `torch`, `torchaudio`, and
-`torchvision` instead of `rocm[libraries,devel]`:
+`torchvision` instead of `rocm[libraries,devel]`. Specfic versions should be
+specified to avoid incompatibilities which might arise due to `pip`'s default
+behavior. For example, for `torch` 2.7.1:
 
 #### gfx94X-dcgpu
 
 ```bash
 python -m pip install \
   --index-url https://rocm.nightlies.amd.com/v2/gfx94X-dcgpu/ \
-  torch torchaudio torchvision
+  torch==2.7.1 torchaudio==2.7.1a0 torchvision==0.22.1
 ```
 
 #### gfx950-dcgpu
 
 ```bash
 python -m pip install \
   --index-url https://rocm.nightlies.amd.com/v2/gfx950-dcgpu/ \
-  torch torchaudio torchvision
+  torch==2.7.1 torchaudio==2.7.1a0 torchvision==0.22.1
 ```
 
 #### gfx110X-dgpu
 
 ```bash
 python -m pip install \
   --index-url https://rocm.nightlies.amd.com/v2/gfx110X-dgpu/ \
-  torch torchaudio torchvision
+  torch==2.7.1 torchaudio==2.7.1a0 torchvision==0.22.1
 ```
 
 #### gfx1151
 
 ```bash
 python -m pip install \
   --index-url https://rocm.nightlies.amd.com/v2/gfx1151/ \
-  torch torchaudio torchvision
+  torch==2.7.1 torchaudio==2.7.1a0 torchvision==0.22.1
 ```
 
 #### gfx120X-all
 
 ```bash
 python -m pip install \
   --index-url https://rocm.nightlies.amd.com/v2/gfx120X-all/ \
-  torch torchaudio torchvision
+  torch==2.7.1 torchaudio==2.7.1a0 torchvision==0.22.1
 ```
 
+Similarly, compatible wheels for `torch 2.9.0` can be installed with 
```

Comment:
This line has trailing whitespace so it fails the pre-commit (https://pre-commit.com/) check: https://github.com/ROCm/TheRock/actions/runs/17446984142/job/49543743437?pr=1385. I'm adding notes on pre-commit to our CONTRIBUTING.md in https://github.com/ROCm/TheRock/pull/1386